src/System.Management.Automation/engine/LanguagePrimitives.cs

```diff
@@ -4243,8 +4243,8 @@ public object Invoke(object valueToConvert, Type resultType, bool recurse, PSObj
             }
         }
 
-        private static Dictionary<ConversionTypePair, ConversionData> s_converterCache
-            = new Dictionary<ConversionTypePair, ConversionData>(256);
+        private static Dictionary<ConversionTypePair, ConversionData> s_converterCache = new Dictionary<ConversionTypePair, ConversionData>(256);
+        private static ConversionData<object> s_noConversion = new ConversionData<object>(ConvertNoConversion, ConversionRank.None);
```

Comment:
I'm not sure it's worth caching this instance. It should be rarely needed and multiple instances are fine.

Reply:
My bad. This static field is not used anywhere. I will remove it.